index.md

```diff
@@ -11,43 +26,58 @@ import CSV files, the structure of data.frame, how to deal with factors, how to
 add/remove rows and columns, and finish with how to calculate summary statistics
 for each level and a very brief introduction to plotting.
 
-> ## Prerequisites
->
-> * Having RStudio installed
 
-## Topics
+**Content Contributors: {{page.contributors | join: ', ' %}}**
+
+
+**Lesson Maintainers: {{page.maintainers | join: ', ' %}}**
+
+
+**Lesson status: {{ page.status }}**
+
+<!--
+  [Information on Lesson Status Categories]()
+-->
+
+<!-- ###### INDEX OF LESSONS ON THIS TOPIC ###### -->
+
+## Lessons:
+
+1. [Lesson 00 Before we start](00-before-we-start.html)
+2. [Lesson 01 Introduction to R](01-intro-to-R.html)
+3. [Lesson 02 Starting with data](02-starting-with-data.html)
+4. [Lesson 03 Introducing `data.frame`](03-data-frames.html)
+5. [Lesson 04 Aggregating and analyzing data with dplyr](04-dplyr.html)
+6. [Lesson 07 Data visualisation with ggplot2](05-visualisation-ggplot2.html) 
+7. [Lesson 06 R and SQL](06-r-and-sql.html)
+
+
+## Data
+
+Data files for the lesson are available here: ({{page.dataurl %}})[{{page.dataurl %}}]
 
-* [Before we start](00-before-we-start.html)
-* [Introduction to R](01-intro-to-R.html)
-* [Starting with data](02-starting-with-data.html)
-* [The `data.frame` class](03-data-frames.html)
-* [Aggregating and analyzing data with dplyr](04-dplyr.html)
 
-## Other resources
+### Requirements
 
-* [Motivation](motivation.html)
+Data Carpentry's teaching is hands-on, so participants are encouraged to use
+their own computers to insure the proper setup of tools for an efficient workflow.
+*These lessons assume no prior knowledge of the skills or tools*, but working
+through this lesson requires working copies of the software described below.
+To most effectively use these materials, please make sure to install everything
+*before* working through this lesson.
 
-## Organization of the repository
+{% if page.software == "Python" %}
+{% include pythonSetup.html %}
+{% elsif page.software == "Spreadsheets" %}
+{% include spreadsheetSetup.html %}
+{% elsif page.software == "R" %}
+{% include rSetup.html %}
+{% else %}
+{% include anySetup.html %}
+{% endif %}
 
-The lessons are written in Rmarkdown. A Makefile generates an html page for each
-topic using knitr. In the process, knitr creates an intermediate markdown
-file. These are removed by the Makefile to avoid clutter.
+<p><strong>Twitter</strong>: @datacarpentry</p>
 
-The Makefile also generates a "skeleton" file that is intended to be distributed
```

Comment:
Yes, that works. The Makefile would need to updated though, as currently README and index are kept in sync by it.